pelicun/settings/schema.json

```diff
@@ -0,0 +1,451 @@
+                      "type": "boolean"
+                    }
+                  }
+                },
+                "Damage": {
+                  "type": "object",
+                  "properties": {
+                    "Sample": {
+                      "type": "boolean"
+                    },
+                    "Statistics": {
+                      "type": "boolean"
+                    },
+                    "GroupedSample": {
+                      "type": "boolean"
+                    },
+                    "GroupedStatistics": {
+                      "type": "boolean"
+                    }
+                  }
+                },
+                "Loss": {
+                  "type": "object",
+                  "properties": {
+                    "Repair": {
+                      "type": "object",
+                      "properties": {
+                        "Sample": {
+                          "type": "boolean"
+                        },
+                        "Statistics": {
+                          "type": "boolean"
+                        },
+                        "GroupedSample": {
+                          "type": "boolean"
+                        },
+                        "GroupedStatistics": {
+                          "type": "boolean"
+                        },
+                        "AggregateSample": {
+                          "type": "boolean"
+                        },
+                        "AggregateStatistics": {
+                          "type": "boolean"
+                        }
+                      }
+                    }
+                  }
+                },
+                "Format": {
+                  "type": "object",
+                  "properties": {
+                    "CSV": {
+                      "type": "boolean"
+                    },
+                    "JSON": {
+                      "type": "boolean"
+                    }
+                  }
+                },
+                "Settings": {
+                  "type": "object",
+                  "properties": {
+                    "CondenseDS": {
+                      "type": "boolean"
+                    },
+                    "SimpleIndexInJSON": {
+                      "type": "boolean"
+                    },
+                    "AggregateColocatedComponentResults": {
+                      "type": "boolean"
+                    }
+                  }
+                }
+              }
+            },
+            "Options": {
+              "type": "object",
+              "properties": {
+                "ListAllDamageStates": {
+                  "type": "boolean"
+                }
+              }
+            },
+            "ApplicationData": {
+              "type": "object",
+              "properties": {
+                "ground_failure": {
+                  "type": "boolean"
+                }
+              },
+              "required": [
+                "ground_failure"
```

Comment:
It's strange to have this as a required property. I think it would be better to have it default to `False` and not require it.

Reply:
I change this in a future commit.

Beyond that specific point, we can document default values in the JSON schema, but we currently only use it for validation. Do you think it would be worthwhile to document all defaults?

Reply:
I don't think adding defaults to the schema should be prioritized now.

Perhaps we could add this a new issue and tag it as a desirable, but not critical feature for the future.

Reply:
I created issue #55. I can't assign labels to issues.